tests/http/requirements.txt

```diff
@@ -28,3 +28,4 @@ cryptography
 multipart
 websockets
 psutil
+httpx
```

Comment:
I'm wondering about this, where is `httpx` used by pytest?

CI uses this requirement.txt to install deps, and works without.